### ISSUE TITLE
Android Java Fundamentals codelab: 

### ISSUE THREAD

**Issue by danserboi** [google-developer-training/android-fundamentals-apps-v2#174]
**Describe the problem**
String variable spinnerLabel in onItemSelected method is not previously declared.

**In which lesson and step of the codelab can this issue be found?**
Lesson number: 04.2: Input controls.
Step number: 3.3 Add code to respond to Spinner selections.

**Additional information**
The correct version is the following:

public void onItemSelected(AdapterView<?> adapterView, View view, int
               i, long l) {
   String spinnerLabel = adapterView.getItemAtPosition(i).toString();
   displayToast(spinnerLabel);
}

**codelab:** android-fundamentals

**Screenshot:**
![3 3_String_variable_undeclared](https://user-images.githubusercontent.com/57451294/89780708-a578a000-db1a-11ea-8342-087f41f58600.png)

**Comment by shahzadansari** [comment]
Yes this seems to be the problem in the  [Android fundamentals 04.2: Input Controls](https://codelabs.developers.google.com/codelabs/android-training-input-controls/index.html?index=..%2F..%2Fandroid-training#5). But the [*solution code*](https://github.com/google-developer-training/android-fundamentals-apps-v2/blob/master/DroidCafeInput/app/src/main/java/com/example/android/droidcafeinput/OrderActivity.java)  for [DroidCafeInput](https://github.com/google-developer-training/android-fundamentals-apps-v2/tree/master/DroidCafeInput/app/src/main/java/com/example/android) contains the correct code for *spinnerLabel* String.
![screenshot_20200811_123905](https://user-images.githubusercontent.com/43310446/89870874-f1d1e780-dbcf-11ea-94c7-34967f1b9595.png)
